### DISCUSSION TITLE
无法打开视频简介中的链接，点击“在浏览器打开”崩溃

### DISCUSSION THREAD

**Discussion by SuanOrange** [ywmoyue/biliuwp-lite#591] [Q&A] [answered]
### Bug 描述

无法打开视频简介中的链接，显示“出现未知错误”，点击在浏览器打开，客户端崩溃

### 操作步骤

```plain text
1.打开应用(版本为v4.6.29.0)
2.打开一个简介有链接的视频
3.点击链接，出现“出现未知错误，已记录”
4.点击右上角“使用浏览器打开”，客户端崩溃
```

### 期望结果

可以正常显示链接中的内容，可以由客户端正常跳转浏览器

### 截图

https://github.com/ywmoyue/biliuwp-lite/assets/67503405/0f4e8093-5a18-4d1f-a670-84ac19adbe36

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 系统版本

其它 (需注明)

### 其它系统版本

Windows 10 22H2 (Build 19045)

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
我这边测了是正常的，你需要检查下系统自带的WebView2是否工作正常

可以应用商店安装`WinUI 2 Gallery`, 在该应用中搜索打开WebView2看该控件是否能打开一个页面

![image](https://github.com/ywmoyue/biliuwp-lite/assets/28388300/9568e6a9-ac33-42b5-8f1d-76c9fbc10ef8)

**Reply by SuanOrange** [reply]
![Snipaste_2024-04-22_12-08-31](https://github.com/ywmoyue/biliuwp-lite/assets/67503405/d79440be-2192-40d6-bb89-db272260a685)
无法显示，请问该如何解决这个问题？

**Reply by ywmoyue** [reply, marked as answer]
https://developer.microsoft.com/zh-cn/microsoft-edge/webview2?form=MA13LH#download-section

不知道这个是不是，你试试
![image](https://github.com/ywmoyue/biliuwp-lite/assets/28388300/54f02f4b-3b97-4070-a42c-f0e89a6a29f3)

**Reply by SuanOrange** [reply]
已解决，感谢
